test/internal/SetupResumeAcceptorTest.cpp

```diff
@@ -256,8 +256,7 @@ TEST(SetupResumeAcceptor, EventBaseDisappear) {
   auto evb = std::make_unique<folly::EventBase>();
 
   SetupResumeAcceptor acceptor{
-      ProtocolVersion::Unknown, evb.get(), std::this_thread::get_id()};
-  evb.reset();
+      ProtocolVersion::Unknown, evb.get()};
```

Comment:
Example usage of the `owner_` field in action, without it the `evb.reset()` will cause `~SetupResumeAcceptor` to crash.

Reply:
It looks like `owner_` is supposed to be an optional field, but there's no usages where it's optional. Is there ever a case where we want to be able to destroy the EB before destroying the SetupResumeAcceptor? We already break if the user destroys an RSocketServer's EB before the closing it.

Reply:
Yeah, the field only really fixes things for `SetupResumeAcceptor`, but `RSocketServer` is still hit by it.

We can probably remove this now after my recent `HttpConnectionAcceptor` changes in RTGW.